proto/buf/alpha/registry/v1alpha1/labels.proto

```diff
@@ -73,10 +78,26 @@ message MoveLabelResponse {}
 message GetLabelsRequest {
   string repository_owner = 1;
   string repository_name = 2;
+  // Optional label_name filter, in case you want to get labels in a namespace+name.
```

Comment:
Where does the `namespace` come from for this request?

Reply:
Good question, it was also confusing for me at first. `message LabelName` [contains both namespace and name](https://github.com/bufbuild/buf/blob/cb7982633f38fa222e1141fcd9b04d004faaf1e9/proto/buf/alpha/registry/v1alpha1/labels.proto#L35-L38).

Reply:
Ohh... I see